build/openwisp_freeradius/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM freeradius/freeradius-server:3.0.19-alpine
+
+RUN apk add --no-cache --update postgresql-dev postgresql-client && \
+    rm -rf /var/cache/apk/* /tmp/*
+
+ENTRYPOINT ["sh", "init_command.sh"]
+EXPOSE 1812/udp 1813/udp
+
+COPY ./common/init_command.sh \
+    ./common/utils.sh ./
+COPY --chown=root:root ./openwisp_freeradius/raddb/ /etc/raddb/
+
+ENV MODULE_NAME=freeradius \
+    DASHBOARD_DOMAIN=dashboard.example.com \
```

Comment:
Does this mean the freeradius container will attempt to connect to the Rest API on the dashboard container?  Shouldn't it connect to the openwisp_radius container?

Reply:
Oh yes, the name here makes it all confusing, I am changing the variable name to `RADIUS_API_DOMAIN`.
Since, Freeradius can work with both dashboard and radius and I am using dashboard for testing, that's why it was dashboard here.
Yes, the traffic is going through nginx because the containers are running `uwsgi` which is unsupported protocol for freeradius.